### ISSUE TITLE
Common service connection issue forinvoke pipeline task

### ISSUE THREAD

**Issue by mvbugge** [microsoft/dstoolkit-mlops-base#72]
In the [pipeline azure-pipelines/templates/utils/invoke-aml-pipeline.template.yml](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/azure-pipelines/templates/utils/invoke-aml-pipeline.template.yml)

`- job: invoke_pipeline
    displayName: ${{parameters.jobDisplayName}}
    pool: server
    timeoutInMinutes: 0
    dependsOn: pipeline_id
    variables: 
      pipeline_id: $[ dependencies.pipeline_id.outputs['get_pipeline_id.pipeline_id'] ]
    steps:
        - task: ms-air-aiagility.vss-services-azureml.azureml-restApi-task.MLPublishedPipelineRestAPITask@0
          displayName: 'Invoke AML Pipeline'
          inputs:
            azureSubscription: '${{parameters.serviceConnection}}'
            PipelineId: '$(PIPELINE_ID)'
            ExperimentName: '${{parameters.experimentName}}'`

Reading online, and from my own experiences, the task will not run unless given a specific machine learning workspace service principal connection in DevOps. A typical service principal connection will not suffice.

**Comment by mariamedp** [comment]
That's correct. When invoking an AML pipeline, the variable `SERVICECONNECTION_WS` is used instead of `SERVICECONNECTION_RG` (both are defined in the [infra configuration file](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/configuration/configuration-infra-DEV.variables.yml)). For example in the modeling pipeline: https://github.com/microsoft/dstoolkit-mlops-base/blob/322f45171b91646eae8b4d276c371d3685d7c54f/azure-pipelines/PIPELINE-1-modeling.yml#L97

If the wrong service connection is used, the task will fail with no explanation. That issue is documented in the FAQ [here](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/docs/how-to/FAQ.md#when-running-the-ado-training-pipeline-the-pipeline-fails-at-the-invoke-step-whats-the-error-).

I'll close the issue now, thanks for creating it so people will find the solution in the future.
